views/ticket/index.pug

```diff
@@ -32,10 +32,9 @@ block content
           if user.role === "ADMIN"
             form(id=`ticket-form-${ticket.id}` class='flex flex-row items-center mb-2 space-x-3')
```

Comment:
I think we can clean up some of these classes here, e.g. we don't need flex or spacing utilities anymore. We can also remove the div below.